src/Provider/3dTilesProvider.js

```diff
@@ -155,11 +178,35 @@ export function getObjectToUpdateForAttachedLayers(meta) {
     }
 }
 
+/** @classdesc
+ * Class for $3dtilesLayer.
+ *
+ * @example
+ * var layer = new itowns.GeometryLayer('3dtiles-example');
+ * layer.protocol = '3d-tiles'
+ * layer.url = 'http://example/tileset.json';
+ *
+ * @property {string} url - tileset.json URL
+ * @property {boolean|THREE.Material} [overrideMaterial=false] - override meshes material
+ * embedded in the binary files and use a default one instead.
+ * @property {THREE.Material} [material=THREE.PointsMaterial] - material cloned
+ * and assigned each time a points mesh is created.
+ * @property {number} [sseThreshold=16] - s(creen) s(pace) e(rror) threshold in pixels.
+ * Define how many pixels the geometricError from a tile must cover to kick the
+ * subdivision mechanism in.
+ * @property {number} [cleanupDelay=1000] - delay in milliseconds after which an
+ * undisplayed tiles will be removed from memory.
+ * @property {Function} onAddContent - callback called when new content is added.
+ */
+export class $3dtilesLayer {
+}
+
 function preprocessDataLayer(layer, view, scheduler) {
     layer.preUpdate = layer.preUpdate || pre3dTilesUpdate;
     layer.update = layer.update || process3dTilesNode();
     layer.sseThreshold = layer.sseThreshold || 16;
     layer.cleanupDelay = layer.cleanupDelay || 1000;
+    layer.onAddContent = layer.onAddContent || (() => {});
```

Comment:
I wouldn't use this name: it seems like a callback to be trigger after a function named `addContent`. What about something like `onContentAdded` or `onContentLoaded` ?

Reply:
ok to `onContentAdded`

Reply:
Maybe `onTileContentLoaded` to be consistent with the standard names ?

Reply:
ok for `onTileContentLoaded`